sources/features/gsub.fea

```diff
@@ -0,0 +1,752 @@
+     language CRT  exclude_dflt;
+      lookup lnum_lining_figures;
+     language TRK  exclude_dflt;
+      lookup lnum_lining_figures;
+} lnum;
+
+feature pnum {
+
+ script DFLT;
+     language dflt ;
+      lookup pnum_proportional_figures;
+
+ script cyrl;
+     language dflt ;
+      lookup pnum_proportional_figures;
+
+ script grek;
+     language dflt ;
+      lookup pnum_proportional_figures;
+
+ script latn;
+     language dflt ;
+      lookup pnum_proportional_figures;
+     language AZE  exclude_dflt;
+      lookup pnum_proportional_figures;
+     language CRT  exclude_dflt;
+      lookup pnum_proportional_figures;
+     language TRK  exclude_dflt;
+      lookup pnum_proportional_figures;
+} pnum;
+
+feature tnum {
+
+ script DFLT;
+     language dflt ;
+      lookup tnum_tabular_figures;
+
+ script cyrl;
+     language dflt ;
+      lookup tnum_tabular_figures;
+
+ script grek;
+     language dflt ;
+      lookup tnum_tabular_figures;
+
+ script latn;
+     language dflt ;
+      lookup tnum_tabular_figures;
+     language AZE  exclude_dflt;
+      lookup tnum_tabular_figures;
+     language CRT  exclude_dflt;
+      lookup tnum_tabular_figures;
+     language TRK  exclude_dflt;
+      lookup tnum_tabular_figures;
+} tnum;
+
+feature zero {
+
+ script DFLT;
+     language dflt ;
+      lookup zero_slash;
+
+ script cyrl;
+     language dflt ;
+      lookup zero_slash;
+
+ script grek;
+     language dflt ;
+      lookup zero_slash;
+
+ script latn;
+     language dflt ;
+      lookup zero_slash;
+     language AZE  exclude_dflt;
+      lookup zero_slash;
+     language CRT  exclude_dflt;
+      lookup zero_slash;
+     language TRK  exclude_dflt;
+      lookup zero_slash;
+} zero;
+
+feature salt {
+
+ script DFLT;
+     language dflt ;
+      lookup salt_stylistic_alternatives;
+
+ script cyrl;
+     language dflt ;
+      lookup salt_stylistic_alternatives;
+
+ script grek;
+     language dflt ;
+      lookup salt_stylistic_alternatives;
+
+ script latn;
+     language dflt ;
+      lookup salt_stylistic_alternatives;
+     language AZE  exclude_dflt;
+      lookup salt_stylistic_alternatives;
+     language CRT  exclude_dflt;
+      lookup salt_stylistic_alternatives;
+     language TRK  exclude_dflt;
+      lookup salt_stylistic_alternatives;
+} salt;
+
+feature ccmp {
+
+ script DFLT;
+     language dflt ;
+      lookup ccmp_contextual;
+
+ script cyrl;
+     language dflt ;
+      lookup ccmp_contextual;
+
+ script grek;
+     language dflt ;
+      lookup ccmp_contextual;
+
+ script hebr;
+     language dflt ;
+      lookup ccmp_contextual;
+
+ script latn;
+     language dflt ;
+      lookup ccmp_contextual;
+     language AZE  exclude_dflt;
+      lookup ccmp_contextual;
+     language CRT  exclude_dflt;
+      lookup ccmp_contextual;
+     language TRK  exclude_dflt;
+      lookup ccmp_contextual;
+} ccmp;
+
+feature ss01 {
+  featureNames {
+    name 3 1 0x409 "Low diaeresis on 'A', 'E', 'O'";
```

Comment:
The shorter `name "Low diaeresis on 'A', 'E', 'O'";` should work.

Reply:
It did work.